wordpress-mobile/WordPress-iOS
# Fix compliance popover breaking other views presentation

Fixes #21758

## Description
This PR fixes an issue where the Compliance Popover presentation could break other screens presentation.

## Test Instructions

1. Make sure you're in compliant country or  `shouldShowPrivacyBanner` returns `true`
2. Fresh install WordPress/Jetpack app
3. Log in
4. Click "Checking Stats" on onboarding questions
5. **Verify** the compliance popover appears without breaking Stats navigation

## Regression Notes

1. Potential unintended areas of impact
Compliance popover presentation.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
